app/src/main/java/com/fabulouszanna/pokedex/model/pokemontypes/PokemonType.kt

```diff
@@ -0,0 +1,81 @@
+package com.fabulouszanna.pokedex.model.pokemontypes
+
+sealed class PokemonType(
```

Comment:
I see you're going the sealed class route, cool!

I would suggest using named arguments to improve clarity. It'll be more verbose.

Put the coursor right after `PokemonType(` and push `ALT+Enter`. There should be a quick option to add argument names.

Reply:
Addressed in next commit